src/algebra/group/GroupModule.lean

```diff
@@ -21,10 +37,16 @@ restate_axiom hom.comm'
 
 namespace hom
 
+/-- The identity morphism on a `GroupModule G`. -/
 @[simps]
 def id (M : GroupModule G) : GroupModule.hom M M :=
 { hom := 𝟙 M.V }
 
+instance (M : GroupModule G) : inhabited (GroupModule.hom M M) := ⟨id M⟩
```

Comment:
@gebner Which instance do you want here?
Another option is to give the `0`-morphism between any `M` and `N`...

Reply:
I think the linter demanding these sorts of instances is not a good idea. If the structure has type parameters, there's no point making the linter happy by constructing instances for particular values of those parameters...